build/lender/Lender.sol

```diff
@@ -43,6 +43,23 @@ contract Lender {
         feenominator = 1000;
     }
 
+    /// @notice Allows the admin to bulk approve given compound addresses at the underlying token, saving marginal approvals
+    /// @param u array of underlying token addresses
+    /// @param c array of compound token addresses
```

Comment:
"compound token addresses" wut? 

we are ridding swivel of compound-specific hard coded stuff (so i doubt we wanna bring that same thing into illuminate). shouldn't `c` be something more generic here than specifically _Compound_ ?

"Compounding Token" would be a swivel-esque (v3) term...

Reply:
We can go with that. This was directly pulled from the swivel codebase I believe.